src/components/HoverHint/HoverHint.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import styles from './HoverHint.scss';
+
+const HoverHint = ({ isCardVideo, showPreview, canPlay }) =>
```

Comment:
| my opinion |
You can write an interface for HoverHint to keep passed properties instead of passing them directly.